LOCALAPP.md

```diff
@@ -12,8 +12,8 @@ These are some of the required steps, tips and tricks when it comes to running a
 2. Newest [Git](https://git-scm.com/downloads)
 3. A code editor - we like [Visual Studio Code](https://code.visualstudio.com/Download)
     - Also install [recommended extensions](https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions) (f.ex. [C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp))
-4. For Windows/MacOS [Docker Desktop](https://www.docker.com/products/docker-desktop)
-5. Update hosts file (C:/Windows/System32/drivers/etc/hosts) by adding the following values. On MacOS add the same values to values /private/etc/hosts using cmd `sudo nano /private/etc/hosts`.
+4. [Docker Desktop](https://www.docker.com/products/docker-desktop) for Windows/MacOS, or install Docker from your package manager on Linux
```

Comment:
When I did this in February, the version of Ubuntu that my azure VM came preinstalled with, had an old version of docker-compose that did not support the `profile:` functionality, so I had to either edit the compose file to remove it, or install the latest docker from an unofficial repo. Should probably mention the compatibility issue.

https://github.com/Altinn/altinn-studio/pull/8024/files#diff-49542e2257dc85f662593fea7d3e1b67e062e87dbf3acb83d336fdef33f70323R16